### ISSUE TITLE
Suggest to create README and LICENSE file when not exist

### ISSUE THREAD

**Issue by davidak** [go-gitea/gitea#15626]
We at @Codeberg-org would like to suggest to our users to create README and LICENSE files, so the repos are more useful for the community.

https://codeberg.org/Codeberg/Community/issues/293
https://codeberg.org/Codeberg/Community/issues/414

The GitHub design team always does a good job, so i suggest to do it as they to.

![Screenshot from 2021-04-26 17-34-22](https://user-images.githubusercontent.com/91113/116113676-fcc69c80-a6b8-11eb-9e50-07cd9605277e.png)

Show hints for readme and license with option to create them in webui.

Content should contain:

- readme helps viewers to understand what a project is about, how to use it and how to contribute
- an open source license makes it legally possible for people to use the project
- write as Call To Action (like github did)
- keep it short

I'm not a native speaker, so suggestions welcome.

**Comment by jolheiser** [comment]
For what it's worth, if/when implemented I would like it to be configurable. 

I can definitely see why Codeberg would want this, however I can also see where many smaller instances would just see clutter.

**Comment by fnetX** [comment]
Well, actual downstream suggestion was actually to have a way to add a licence via such a convenient menu as you have on repo creation ... but we were also heavily discussing reminders for every bit and piece ...

The first one would already be an improvement without suggesting a specific use case for every self-hosting Gitea users.

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
> For what it's worth, if/when implemented I would like it to be configurable.
> 

always +1 for configurability

**Comment by lunny** [comment]
I agree with that. Well document is a basic of repository whether it is open source. The hint to create a readme is just a hint and when click it to go to the web editor and fill in the `README.md` in path input box.
